src/components/Statistics/AmountPicker.vue

```diff
@@ -16,28 +16,55 @@
         />
       </div>
     </div>
-    <div>
+    <div class="row no-wrap">
       <q-slider
-        :value="value"
+        :value="limitedValue"
         @input="$emit('input', arguments[0])"
         :min="0"
         :max="70"
         :step="0.5"
         label
         snap
       />
+      <!-- don't use type="number" here because browsers might enforce different decimal setting
+       depending on browser locale-->
+      <q-input
+        style="width: 5em; margin: 0 5px 0 2em; text-align: right"
+        v-model="valueToNumber"
+        align="right"
+      />
+      <span style="padding: 3px">kg</span>
     </div>
   </div>
 </template>
 
 <script>
-import { QSlider } from 'quasar'
+import { QInput, QSlider } from 'quasar'
 import AmountViewer from './AmountViewer'
 import AmountBox from './AmountBox'
 
 export default {
   props: { value: { default: 0, type: Number } },
-  components: { QSlider, AmountViewer, AmountBox },
+  components: { QInput, QSlider, AmountViewer, AmountBox },
+  computed: {
+    limitedValue: {
+      get () {
+        return Math.min(70, this.value)
+      },
+      set (v) {
+        this.value = v
+      },
+    },
+    valueToNumber: {
+      get () {
+        return this.value
+      },
+      set (v) {
+        const value = parseFloat(v, 10)
```

Comment:
Just changed parseInt to parseFloat - or was there a specific reason to use Int?

Reply:
by mistake :)